### DISCUSSION TITLE
Creating Internal Navigation Links with Templater

### DISCUSSION THREAD

**Discussion by rwsaltzgiver** [SilentVoid13/Templater#517] [Help]
I have a set of files that are numbered sequentially (think book chapters). I'd like to create a template that reads the chapter number (which is in the frontmatter yaml) and returns links to the files before and after the current file. I've tried using <% tp.frontmatter.chapter %> to return the value and operators (+1, -1) to change the value, but that returns a parsing error. Any thoughts on how to solve this?

**Comment by metruzanca** [comment]
If you meant to say <%+ %> for dynamic templates, which IMO would make more sense for your use case, then sadly this currently isn't possible. I've tried to do this same thing a few times and always disapointed its not possible. I think this is because of how templater processes dynamic templates and that it runs them after the obsidian has parsed the markdown.
